README.md

```diff
@@ -1,291 +1,173 @@
+- **Run a program using the command line.**
+- **Use Ruby syntax for basic programming operations.**
+- **Use Ruby to solve basic coding challenges.**
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
-
-## 🚀 Live Demo <a name="live-demo"></a>
-
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://google.com)
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- GETTING STARTED -->
-
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
-
-To get a local copy up and running, follow these steps.
+To clone and run this application, you'll need [Git](https://git-scm.com) and [Ruby](https://rubyinstaller.org/downloads/) installed on your computer.
 
 ### Prerequisites
+From your command line:
 
-In order to run this project you need:
-
-<!--
-Example command:
+On macOS, by using [Homebrew](https://brew.sh/) you can install.
 
-```sh
- gem install rails
+```bash
+# install rbenv
+$ brew install rbenv
 ```
- -->
 
-### Setup
-
-Clone this repository to your desired folder:
-
-<!--
-Example commands:
-
-```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
+```bash
+# install ruby-build
+$ brew install ruby-build
 ```
---->
 
-### Install
+On Windows, by using [chocolatey](https://chocolatey.org) you can install.
 
-Install this project with:
-
-<!--
-Example command:
-
-```sh
-  cd my-project
-  gem install
+```bash
+# install ruby
+C:\> choco install ruby
 ```
---->
-
-### Usage
 
-To run the project, execute the following command:
 
-<!--
-Example command:
+### Setup
 
-```sh
-  rails server
-```
---->
+Clone this repository to your desired folder:
 
-### Run tests
+```bash
+# Clone this repository
+$ git clone https://github.com/Kgomotso196/Decode-Morse-Message.git
 
-To run tests, run the following command:
+# Go into the repository
+$ cd Decode-Morse-Message
 
-<!--
-Example command:
+# Install dependencies
+$ bundle install
 
-```sh
-  bin/rails test test/models/article_test.rb
 ```
---->
 
-### Deployment
+### Usage
 
-You can deploy this project using:
+After you clone this repo to your desktop and install the dependencies, go to its root directory and run `ruby decode_morse_code.rb` to run the application.
 
-<!--
-Example:
 
-```sh
-
-```
- -->
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- AUTHORS -->
 
 ## 👥 Authors <a name="authors"></a>
 
-> Mention all of the collaborators of this project.
-
-👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+👤 **Author**
 
-👤 **Author2**
+- GitHub: [@yemidada](https://github.com/yemidada)
+- Twitter: [@naijavine](https://twitter.com/naijavine)
+- LinkedIn: [yemidada](https://linkedin.com/in/yemidada)
```

Comment:
- [ ] Please kindly fix the twitter account link, I got a message that the account does not exist. If you don't have a working link at the moment, I think it's better to remove it in order to have an engaging README file. ℹ️